LICENSE

```diff
@@ -1,56 +1,30 @@
-
-You can redistribute Prawn and/or modify it under either the terms of the GPLv2
-or GPLv3 (see GPLv2 and GPLv3 files), or the conditions below:
-
-  1. You may make and give away verbatim copies of the source form of the
-     software without restriction, provided that you duplicate all of the
-     original copyright notices and associated disclaimers.
-
-  2. You may modify your copy of the software in any way, provided that
-     you do at least ONE of the following:
-
-       a) place your modifications in the Public Domain or otherwise
-          make them Freely Available, such as by posting said
-    modifications to Usenet or an equivalent medium, or by allowing
-    the author to include your modifications in the software.
-
-       b) use the modified software only within your corporation or
-          organization.
-
-       c) rename any non-standard executables so the names do not conflict
-    with standard executables, which must also be provided.
-
-       d) make other distribution arrangements with the author.
-
-  3. You may distribute the software in object code or executable
-     form, provided that you do at least ONE of the following:
-
-       a) distribute the executables and library files of the software,
-    together with instructions (in the manual page or equivalent)
-    on where to get the original distribution.
-
-       b) accompany the distribution with the machine-readable source of
-    the software.
-
-       c) give non-standard executables non-standard names, with
-          instructions on where to get the original software distribution.
-
-       d) make other distribution arrangements with the author.
-
-  4. You may modify and include the part of the software into any other
-     software (possibly commercial). 
-
-  5. The scripts and library files supplied as input to or produced as 
-     output from the software do not automatically fall under the
-     copyright of the software, but belong to whomever generated them, 
-     and may be sold commercially, and may be aggregated with this
-     software.
-
-  6. THIS SOFTWARE IS PROVIDED "AS IS" AND WITHOUT ANY EXPRESS OR
-     IMPLIED WARRANTIES, INCLUDING, WITHOUT LIMITATION, THE IMPLIED
-     WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
-     PURPOSE.
+Prawn may be used under any of the following licenses:
+
+1. The Prawn License (see: LICENSE-PRAWN), which is similar to
+   Matz's original licensing terms for Ruby.
+2. GPLv2 (see: LICENSE-GPLv2)
+3. GPLv3 (see: LICENSE-GPLv3)
+
+The maintainers of this repository plan to migrate to the MIT License
+(see: LICENSE-MIT) in the future. Effective November 8, 2023, by
+contributing to this repository, you consent to license under the MIT License
+your contribution together with any other contributions to this repository,
+its forks, and other repositories in the "prawnpdf" GitHub organization,
+in addition to the existing license(s). This includes without limitation:
```

Comment:
Do you mean that contributions done, for example, **only** to prawnpdf/prawn should fall under the MIT?

Reply:
Hmm I've dialed back the language for now. Basically, I'd like to ensure previous Prawn contributors have opted-in to MIT.

I think what I will do instead is make a checkbox in the Issue/PR template something like `[ ] I agree for my Prawn contributions (including any other contributions) to be licensed under the MIT License.`

Reply:
I think you would need to differentiate between code for this repo and all other code written by the person which might include code for Prawn.

So maybe something like `[ ] My contribution is licensed under the MIT license.` (just for this repo) and `[ ] I agree to license all my contributions that are/will be included in this repo under the MIT license.` (for all code from that one person).

Reply:
Yep that's exactly what I was thinking.